flags/flags.ts

```diff
@@ -262,7 +262,6 @@ export function parseFlags<
 
         if (negate) {
           flags[propName] = false;
-          return;
```

Comment:
I think the `return` was correect here. The issue is that `negate` is defined outside of the loop. Because of this `negate` is never resetted to `false`. We should define the `negate` option inside the loop.

Reply:
Alright, having a second look.

Reply:
Working, better fix, cheers!